q2_feature_table/plugin_setup.py

```diff
@@ -117,6 +117,24 @@
                 "sample."
 )
 
+plugin.methods.register_function(
+    function=q2_feature_table.transpose,
+    inputs={'table': FeatureTable[Frequency]},
+    parameters={},
+    outputs=[('transposed_feature_table',
+               FeatureTable[Frequency])],
+    input_descriptions={
+        'table': 'The feature table to be transposed.'
+    },
+    parameter_descriptions={},
+    output_descriptions={
+        'transposed_feature_table': ('The resulting transposed feature table.')
+    },
+    name='Transpose a feature table.',
+    description='Transpose the rows and columns of a feature table.'
```

Comment:
I was stewing on this a bit - what do you think of:

> Transpose the rows and columns (samples and features) of a feature table.

Reply:
:+1: maybe add: `(typically samples and features)` in the parenthetical?